scipy/ndimage/_ni_support.py

```diff
@@ -95,3 +96,24 @@ def _get_output(output, input, shape=None, complex_output=False):
     elif complex_output and output.dtype.kind != 'c':
         raise RuntimeError("output must have complex dtype")
     return output
+
+
+def _check_axes(axes, ndim):
+    if axes is None:
+        return tuple(range(ndim))
+    elif isinstance(axes, Integral):
```

Comment:
There was a pretty long discussion about the best way to do this in gh-7351. It looked like there was a slight preference for using `operator.index`, but I leave that to your discretion.

Reply:
I will check it using `operator.index`